tests/e2e/user-anonymous.spec.js

```diff
@@ -0,0 +1,79 @@
+import { expect, test } from '@playwright/test';
+
+import orchestrator from 'tests/orchestrator';
+
+import { HomePage } from './page-object/home-page';
+import { LoginPage } from './page-object/login-page';
+import { RecentPage } from './page-object/recent-page';
+import { RegisterPage } from './page-object/register-page';
+
+const titleHome = 'TabNews: Conteúdos para quem trabalha com Programação e Tecnologia';
+
+test.beforeAll(async () => {
+  await orchestrator.waitForAllServices();
+  await orchestrator.dropAllTables();
+  await orchestrator.runPendingMigrations();
+});
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/');
+});
+
+test('should be able to login', async ({ page }) => {
+  let homePage = new HomePage(page);
+
+  let title = await homePage.getTitle();
+  await expect(title).toBe(titleHome);
```

Comment:
Se isso for apenas para aguardar a página carregar, como perguntei em outro comentário, então realizando as alterações propostas a verificação do `title` pode ficar em um único teste, e os outros testes usarão apenas o método para aguardar o carregamento.

E, se isso for feito, não precisamos mais da variável `titleHome`.